src/bot/lib/crud.js

```diff
@@ -0,0 +1,32 @@
+// =============
+// CRUD HELPERS
+// =============
+
+// Add item (skills or categories) with a given name.
+export function addItemByName(token, name, queryFunction) {
+  return queryFunction({token, name});
+}
+
+// processes adds and returns output for dialogue
+export function prepareAddOutput(matchResults, token, queryFunction) {
```

Comment:
I don't feel that the name for this function adequately describes what it does. What if the "try using the list command..." message was wrapped up in a function, and the actual inserting + success message wasn't in a reused function, but was just coded directly into the command?